### ISSUE TITLE
How to change ksc configuration options?

### ISSUE THREAD

**Issue by brunchboy** [valery1707/kaitai-maven-plugin#7]
Because of a bug in the `close()` implementation of `ByteBufferKaitaiStream` I need to be able to run `ksc` with the option `--java-from-file-class io.kaitai.struct.RandomAccessFileKaitaiStream`. I can’t figure out how to do this using the Maven plugin.

**Comment by valery1707** [comment]
Good question.
In next version I add support for this options:
* `--java-from-file-class <class>`: Java class to be invoked in fromFile() helper (default: io.kaitai.struct.ByteBufferKaitaiStream)
* `--opaque-types <value>`: opaque types allowed, default: false
* `--ksc-exceptions`: ksc throws exceptions instead of human-readable error messages

**Comment by brunchboy** [comment]
Great, thank you!

**Comment by valery1707** [comment]
Also can you describe the bug in `ByteBufferKaitaiStream#close`?

**Comment by brunchboy** [comment]
Certainly. It fails to close the underlying `mmap` file, meaning that you cannot eject removable media that has been used with a `ByteBufferKaitaiStream` until you terminate the JVM, so you basically cannot use it with removable media. They have done all they can, the underlying issue is a problem that neither Sun nor Oracle has been able to figure out a solution for since 2004. https://github.com/kaitai-io/kaitai_struct/issues/497

**Comment by valery1707** [comment]
Version `0.1.3` was relased now, after some time it will be accessible in maven central
